wiki/Beatmapping/en.md

```diff
@@ -1,337 +1,229 @@
     -   Linear mode is useful if you want sharp (e.g. 90 degree) corners in your sliders.
 
-Now you can begin adding a slider. Left-click to start the slider, add waypoints with further left-clicks, and terminate with a right-click to make a bezier slider. If you want to create a liner slider, make a bezier-shaped slider first, then left-clicking on an existing vertex will turn that vertex red and create a new anchor point to allow for sharp turns and new curves. It is recommended to make sure the length of the slider is correct before adjusting the shape.
+Now you can begin adding a slider. `Left-click` to start the slider, add waypoints with further `left-clicks`, and terminate with a `right-click` to make a bezier slider. If you want to create a liner slider, make a bezier-shaped slider first, then `left-click` on an existing vertex to turn that vertex red and create a new anchor point to allow for sharp turns and new curves. It is recommended to make sure the length of the slider is correct before adjusting the shape.
 
 Here are some other tips regarding sliders:-
 
 -   To add repeats on a slider, drag the end of the slider in the timeline after finishing placement.
--   Drag an individual waypoint to move it. Right-clicking removes a point, and Ctrl+clicking adds a point.
--   In the same way as hit circles, you can override any snapping by holding shift. This can be useful for aligning sliders perfectly.
+-   Drag an individual waypoint to move it. `Right-click` to remove a point, and `Ctrl` + `Left-click` adds a point.
+-   In the same way as hit circles, you can override any snapping by holding `Shift`. This can be useful for aligning sliders perfectly.
+
+Try and put some effort into making your sliders look nice! Lopsided, wonky, or ugly sliders really ruin the look of a beatmap. Additionally, ensure that your sliders do not overlap one another, the tail end of your slider is always visible, and that the slider path is not obscured in any way - such as unpredictable U-turns or making the head and tail of the slider overlap. These slider-crafting pitfalls should be avoided!
 
-Try and put some effort into making your sliders look nice! Lopsided/overlapped/wonky/plain-ugly sliders really ruin the look of a beatmap! Additionally, ensure that your sliders do not overlap one another, the tail end of your slider is always visible, and that the slider path is not obscured in any way - such as unpredictable U-turns or making the head and tail of the slider overlap. These slider-crafting pitfalls should be avoided!
+### Adding Spinners
 
-Adding Spinners
----------------
+To add a spinner to your map, `left-click` to begin it, then scroll the timeline to where you want the end to be, and terminate with a right-click. Try and keep your spinners to a reasonable length - overly long spinners are tiring and boring, whilst spinners that are too short are not nice. "Spinner spam" (as in, more than 5 spinners in one time) should be avoided!
```

Comment:
L137: Revise: Suggested ``[..] `left-click` to add the starting point for the spinner [..]``